deno-client/mysql.ts

```diff
@@ -0,0 +1,53 @@
+import { createConnection } from "https://deno.land/x/mysql2@v1.0.6/mod.ts";
+import { type Resource } from "./mod.ts";
+
+/**
+ * Establish MySQL connection using MySQL client for Deno:
+ * https://deno.land/x/mysql2@v1.0.6/mod.ts
+ *
+ * IMPORTANT: make sure to close the connection with `.end()`
+ *
+ * @param db the MySQL resource to establish the connection for
+ *
+ * @returns MySQL database connection
+ *
+ * @example
+ * ```ts
+ * const conn = await mysqlConn(db);
+ * await conn.execute('CREATE TABLE IF NOT EXISTS pets (name varchar(255), kind varchar(255))');
+ * await conn.execute('INSERT INTO pets VALUES (?, ?)', ['behemot','cat']);
+ * const [rows] = await conn.execute('SELECT * from pets');
+ * conn.end();
+ * console.log(rows);
+ * ```
+ */
+export async function mysqlConn(db: Resource<"mysql">) {
+    const conn = await createConnection(db);
+    return conn;
+}
+
+/**
+ * Execute SQL query. For more info check:
+ * https://deno.land/x/mysql2@v1.0.6/mod.ts
+ *
+ * @param db the MySQL resource to run sql query for
+ *
+ * @returns array with two items: rows and fields
+ *
+ * @example
+ * ```ts
+ * const [rows, fields] = await mysqlQuery(db)('SELECT * from pets WHERE kind = ?', ['cat']);
```

Comment:
we want signatures of `mysqlQuery` and `pgQuery` to be aligned. meaning:
1.
```diff
- const [rows, fields] = mysqlResult;
+ const { rows, fields } = mysqlResult;
```

2.
```diff
const id = 1337;
- await mySql(db)('SELECT ID, NAME FROM CLIENTS WHERE ID = ?', [id]);
+ await mySql(db)`SELECT ID, NAME FROM CLIENTS WHERE ID = ${id}`;
```